crates/nu-cmd-base/Cargo.toml

```
@@ -11,8 +11,15 @@ version = "0.85.1"
 
 [dependencies]
 nu-engine = { path = "../nu-engine", version = "0.85.1" }
+nu-glob = { path = "../nu-glob", version = "0.85.1" }
 nu-parser = { path = "../nu-parser", version = "0.85.1" }
 nu-path = { path = "../nu-path", version = "0.85.1" }
 nu-protocol = { version = "0.85.1", path = "../nu-protocol" }
+nu-utils = { path = "../nu-utils", version = "0.85.1" }
 indexmap = { version = "2.0" }
 miette = { version = "5.10", features = ["fancy-no-backtrace"] }
+wax = { version = "0.5.0", features = ["diagnostics", "miette"] }
```

```suggestion
wax = { version = "0.6.0", features = ["diagnostics", "miette"] }
```
We've recently upgraded the wax crate